File: spec/python/test_position_to_end.py
import unittest

from position_to_end import PositionToEnd

class TestPositionToEnd(unittest.TestCase):
    def test_position_to_end(self):
        r = PositionToEnd.from_file("src/position_to_end.bin")

        self.assertEqual(r.index.foo, 0x42)
        self.assertEqual(r.index.bar, 0x1234)

File: spec/lua/test_position_to_end.lua
local luaunit = require("luaunit")

require("position_to_end")

TestPositionToEnd = {}

function TestPositionToEnd:test_position_to_end()
    local r = PositionToEnd:from_file("src/position_to_end.bin")

    luaunit.assertEquals(r.index.foo, 0x42)
    luaunit.assertEquals(r.index.bar, 0x1234)
end
